vscode-extensions/snippets/README.md

```diff
@@ -0,0 +1,31 @@
+# Element Framework 7 Snippets
```

Comment:
The copy in this file is a bit wonky

Reply:
This file will be used as a landing page in the VS Code. Do you have any ideas to improve?